homeassistant/components/sensor/alpha_vantage.py

```diff
@@ -158,8 +161,10 @@ def icon(self):
 
     def update(self):
         """Get the latest data and updates the states."""
+        _LOGGER.debug('requesting new data for symbol %s', self._symbol)
```

Comment:
Can you capitalize the first letter in all the comments.

Reply:
Done